doc/changes/changes_2.1.0.md

```diff
@@ -1,11 +1,23 @@
-# Exasol Row Level Security 2.1.0, released 2021-02-11
+# Exasol Row Level Security 2.1.0, released 2021-02-12
 
 Code name: Coverage
 
 Release 2.1.0 of Exasol's Row-Level-Security brings a couple of new administration scripts, that make setting up and maintaining RLS-protected Virtual Schemas easier.
 
 We also now extract the the code coverage from the tests that run inside a docker container. This coverage already existed before, but did not contribute to the static code analysis metrics. The new metrics now reflect the actual coverage situation.
 
+This release also contains a fix for a security issue, classification "medium" in an administration script. The issue can lead to giving unintended row access to users.
+
+You are likely affected if:
+
+* you are using role-based row protection and
+* deleted one or more existing role via an administration script
+
+What can you do to resolve the situation:
+
+* Validate all role masks (column "EXA_ROW_ROLES") on all role-protected tables.
```

Comment:
This won't help the users because we never explain how to do that. Maybe we should add three new admin scripts operating on a role mask: ROLES_WITH_ACCESS, PUBLIC_ROLE_HAS_ACCESS, NONEXISTENT_ROLE_HAS_ACCESS
Then users could select those three scripts for each role-protected table and actually see problems.